### DISCUSSION TITLE
How can i see who created a recipe?

### DISCUSSION THREAD

**Discussion by flomero** [TandoorRecipes/recipes#2742] [Q&A] [answered]
couldn't find it in the UI but the system stores a created_by value

**Comment by smilerz** [comment]
you can see extended attributes in /admin

**Reply by flomero** [reply]
is there any specific reason, this is not visible on the "public" ui?

**Reply by smilerz** [reply]
UI is already overly cluttered and the information is relevent 99% of the time.

**Reply by flomero** [reply]
i thought about something small like that: 
![image](https://github.com/TandoorRecipes/recipes/assets/49640251/872586b1-b0a2-46e2-8f38-18d972a5d429)

and for use case:
- it would be great to see who added the recipe so that you know who you can ask if you have any questions, especially if you are using tandoor with friends and family whom you can contact easily
- if you like a recipe you could search for recipes added by the same user. also realized "created_by" field is missing in search fields

**Reply by smilerz** [reply]
feel free to submit a feature request

**Reply by vabene1111** [reply, marked as answer]
it was in the UI at some point and got lost in some itereation, currently playing around with the new view page with @boppy we will add author back in
